docs/geocode.rst

```diff
@@ -0,0 +1,142 @@
+Geocode
+=======
+
+The ``Geocode`` class provides geocoding using CARTO LDS.
+This process requires you to have a CARTO account with a geocoding provider and geocoding quota assigned, and its use will incurr in the expense of geocoding credits.
+In the case of accounts with soft geocoding limits, additional charges may apply if the monthly quota is exceeded.
+
+The ``geocode`` method provides the interface to geocoding; input data to be geocoded must be provided through a ``Dataset`` or ``DataFrame`` object as the first argument to this method.
```

Comment:
The ```rst
`<cartoframes.data.services.Geocode.geocode>`__ instance method
```
I can't get the rst to display properly in markdown, but it'd be nice if this had the internal sphinx linkage with class/methods.